src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough8.java

```diff
@@ -31,18 +31,18 @@ void methodLastLine(int i) {
 
     void testLastCase(int i) {
         switch (i) {
-            case 0: // violation 'Fall\ through from the last branch of the switch statement'
+            case 0:
                 i++;
                 /* comment */ /* fall thru */ /* comment */
         }
     }
 
     void testLastCase2(int i) {
         switch (i) {
-            case 0:
+            case 0: // violation 'Fall\ through from the last branch of the switch statement'
                 i++;
                 /* comment */ /* comment */
-                /* fall thru */
+
```

Comment:
restore line 45.
if you need violation on such code please add "same" code to new Input

Reply:
Done @romani

Reply:
@aayushRedHat, revert all changes in line 45 and 46.

Reply:
Done @romani